Fix R CMD check error

Bisaloo opened the pull request:
Hi @krisrs1128, do you have any interest getting this package back on CRAN? I'm happy to help you if you need an extra hand.

Bisaloo committed:
Fix R CMD check error

krisrs1128 commented:
Hi @Bisaloo, thank you so much for reaching out. I can confirm that with your pull request the package can pass `R CMD CHECK --as-cran structSSI`. Happy to merge and re-submit to CRAN. Would you like to be added as an author in the `DESCRIPTION`?

PS It's been a long time since I've thought about this code, and I really appreciate your going in and fixing this error.

Bisaloo committed:
Use Authors@R in DESCRIPTION

As required by CRAN for new packages

Bisaloo committed:
Use new CITATION style

As required by CRAN for new packages

Bisaloo committed:
Add ORCiD

As recommended by CRAN for new packages

Bisaloo committed:
Remove Date field

For simplicity and to avoid forgetting updating it while submitting to CRAN.
This field is automatically added by R CMD build.

Bisaloo committed:
Add citation in Description

As likely to be required by CRAN human reviewers during first submission

Bisaloo commented:
> Happy to merge and re-submit to CRAN.

Since previously archived packages are considered as new submissions, there are a couple of extra checks that CRAN is going to perform or human reviewers are going to ask. 

You can reproduce the extra automated checks with `devtools::check(incoming = TRUE)` and most of the extra checks from manual review are documented in [the CRAN cookbook](https://contributor.r-project.org/cran-cookbook/description_issues.html). 

I pushed some additional commits to anticipate these.

> Would you like to be added as an author in the DESCRIPTION?

These are minor packaging fixes so I don't think it's justified but thanks for asking.

krisrs1128 commented:
Thank you @Bisaloo, I appreciate this help. I'll be submitting to CRAN in the next few days. If you change your mind, I'm always happy to add you as an author -- I think what you've done would at least qualify for `contributor` status...
